lib/upperkut/strategies/buffered_queue.rb

```diff
@@ -7,13 +7,65 @@ module Strategies
     class BufferedQueue < Upperkut::Strategies::Base
```

Comment:
What about other strategies?

Reply:
I'll implement `message acknowledgement` support on another PR for the remaining strategies.

These lines keeps everything working fine until there: 
1. https://github.com/ResultadosDigitais/upperkut/pull/62/files#diff-c2d6390b6a15519a1096419828ca1676R142.
2. https://github.com/ResultadosDigitais/upperkut/pull/62/files#diff-ac378525e7716e669f1658b8708e9a4cR73

Reply:
great